portfolio/src/main/webapp/animation.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
```

Comment:
I think you can update the title to be more descriptive.

Additionally, you might want to rename animation.html and animation.css to stars.html and stars.css since they're more descriptive names.